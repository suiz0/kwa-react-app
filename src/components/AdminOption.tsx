import React from 'react';
import {UserAvatarFilled20} from '@carbon/icons-react';
import {
    Link
  } from "react-router-dom";

const AdminOption = (props) => {
    return <UserAvatarFilled20 />
}

export default AdminOption;