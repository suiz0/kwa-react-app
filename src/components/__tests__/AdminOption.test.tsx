import React from 'react';
import AdminOption from '../AdminOption';
import {shallow, configure} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });


let component;

describe('Rendering tests', () => {
    beforeEach(() =>{ component = shallow(<AdminOption />);});

    test('renders without crashing', () => {
        expect(component).toMatchSnapshot();
    });
})