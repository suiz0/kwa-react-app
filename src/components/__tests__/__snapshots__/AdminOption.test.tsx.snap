// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rendering tests renders without crashing 1`] = `<ForwardRef(UserAvatarFilled20) />`;
